src/Runners/PHPUnit/ResultPrinter.php

```diff
@@ -201,6 +202,22 @@ public function printFeedback(ExecutableTest $test): Reader
         return $reader;
     }
 
+    /**
+     * Prints the individual feedback in teamcity form
+     */
+    public function printTeamcityFeedback(ExecutableTest $test): void
```

Comment:
This method must also be tested within `\ParaTest\Tests\Unit\Runners\PHPUnit\ResultPrinterTest`.

Reply:
not relevant, tests for `--log-teamcity` exists